docs/developer-guide/working-with-plugins.md

```diff
@@ -140,6 +140,11 @@ ruleTester.run("custom-plugin-rule", rule, {
 });
 ```
 
+#### Customizing RuleTester
+
+RuleTester internally uses Mocha's `describe` and `it` methods when available to create tests for each valid and invalid case. If you use another test runner, you can override `RuleTester.describe` and `RuleTester.it` to make RuleTester compatible with it and have proper individual tests and feedback.
```

Comment:
Yes, if with “internally” reads better to you as an example of audience for this section, keep it.

Your observation about the link agrees with what I have seen.